ranger/ext/img_display.py

```diff
@@ -686,8 +688,6 @@ def clear(self, start_x, start_y, width, height):
         # kitty doesn't seem to reply on deletes, checking like we do in draw()
         # will slows down scrolling with timeouts from select
         self.image_id -= 1
-        self.fm.ui.win.redrawwin()
-        self.fm.ui.win.refresh()
 
```

Comment:
I see sometimes artifacts in UI, that was probably "fixed" by calling redrawin and refresh here, but I needed to remove it from here because concurrent assess to fm.ui.win caused heavy UI damage.

Where in ranger is the "main" loop? Something that awaits key-presses? Perhaps it would be better to modify it there.